includes/class-monk-links.php

```diff
@@ -499,9 +499,7 @@ public function monk_add_language_author_permalink( $link, $author_id ) {
 		$default_language_url = get_option( 'monk_default_language_url', false );
 		$language             = ( get_query_var( 'lang' ) ) ? get_query_var( 'lang' ) : $this->site_language;
 
-		if ( $this->monk_using_permalinks() ) {
-			$link = $default_language_url || $language !== $default_slug ? str_replace( $this->site_home . '/' . $this->site_root, $this->site_home . '/' . $this->site_root . $language . '/', $link ) : $link;
-		} else {
+		if ( ! $this->monk_using_permalinks() ) {
```

Comment:
Yes it does, a have tested. The author links does not changes with permalinks, it comes the same:
example.com/en/author-name